recipes/blend2d/all/conanfile.py

```diff
@@ -66,9 +61,8 @@ def generate(self):
         tc.variables["BLEND2D_EMBED"] = False
         tc.variables["BLEND2D_STATIC"] = not self.options.shared
         tc.variables["BLEND2D_NO_STDCXX"] = False
+        tc.variables["BLEND2D_EXTERNAL_ASMJIT"] = True
```

Comment:
asmjit as Conan package is now supported without workarounds: https://github.com/blend2d/blend2d/blob/master/CMakeLists.txt#L888